src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java

```diff
@@ -99,17 +99,13 @@ public TABuildWrapper(final String systemProfile, final String versionMajor, fin
 		this.versionMilestone = versionMilestone;
 		this.marker = marker;
 		this.recordSession = recordSession;
+
```

Comment:
new line not needed :)

Reply:
not needed in fact, but nicely separates fields initialization from AppMon client creation